book/website/communication/citable/citable-cff.md

```diff
@@ -0,0 +1,153 @@
+3. If they use the Zotero reference manager, they can import the citation metadata directly from the `CITATION.cff` file in the GitHub repository to their reference manager.
+
+(cm-citable-cff-how-to-create)=
+## How to Create a `CITATION.cff` File (Faruk)
+
+The `CITATION.cff` is a `YAML` file with its own schema definition. The schema defines the rules for each field, and which fields are required and which ones are optional. The user must follow these rules in order to create a valid `CITATION.cff` file.
+
+A minimal example of a valid `CITATION.cff` file, that only contains the required keys, would look like this:
+
+```yaml
+authors:
+  - family-names: John
+    given-names: Doe
+cff-version: 1.2.0
+message: "If you use this software, please cite it using the metadata from this file."
+title: "My research software"
+```
+
+However, adding more fields can help you create more descriptive metadata of your software. The example below also provides important information of software such as version, release date, DOI, license, keywords.
+
+```yaml
+abstract: "This is my awesome research software. It does many things."
+authors:
+  - family-names: John
+    given-names: Doe
+    orcid: "https://orcid.org/0000-0001-8888-9999"
+cff-version: 1.2.0
+date-released: "2021-10-13"
+identifiers:
+  - description: "This is the collection of archived snapshots of all versions of My Research Software"
+    type: doi
+    value: 10.5281/zenodo.123456
+  - description: "This is the archived snapshot of version 0.11.2 of My Research Software"
+    type: doi
+    value: 10.5281/zenodo.123457
+keywords:
+  - "amazing software"
+  - research
+license: Apache-2.0
+message: "If you use this software, please cite it using the metadata from this file."
+repository-code: "https://github.com/citation-file-format/my-research-software"
+title: "My Research Software"
+version: 0.11.2
+```
+
+The complete list of fields is described in the [CFF schema guide](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md). In the next section, you can find out which tools can help you create and use the `CITATION.cff` file.
+
+### Steps to Make Your Software Citable
+
+To make your software citable, you need to follow the three steps below.
+
+#### Step 1. Create a `CITATION.cff` File
+
+There are two ways of creating a `CITATION.cff` file.
+
+1. Edit the file manually in your favorite code editor. The disadvantages of this method are installing the required tools on your system and doing the validation yourself. The error messages of the validation can be relatively long and difficult to understand.
+
+2. Use [cffinit](https://citation-file-format.github.io/cff-initializer-javascript/) which guides you through the process of creating your citation file. The `cffinit` has a few advantages compared to manual editing such as
```

Comment:
Split into two lines.

Comma after closed brackets in the first line?

Reply:
Oh, and I would explain that `cffinit` is web-based at the top. Wasn't clear to me initially.

Reply:
done